src/Cake.Curl.Tests/CurlDownloadMultipleFilesTests.cs

```diff
@@ -501,6 +502,74 @@ public void Should_Not_Set_The_RetryConnRefused_Option_As_Argument()
                 // Then
                 Assert.DoesNotContain("--retry-connrefused", result.Args);
             }
+
+            [Theory]
+            [MemberData(nameof(CurlTimeSpanTestData.TimeSpanData), MemberType = typeof(CurlTimeSpanTestData))]
+            public void Should_Set_The_MaxTime_Option_As_Argument(TimeSpan maxTime)
```

Comment:
I think this test could be simplified by passing the number of seconds as a parameter and create the `TimeSpan` during the _arrange_ phase:

```csharp
[Theory]
[InlineData(0.5)]
[InlineData(60)]
[InlineData(2.99)]
public void Should_Set_The_MaxTime_Option_As_Argument(double seconds)
{
    // Given
    var maxTime = TimeSpan.FromSeconds(seconds);
    ...
}
```

Reply:
@ecampidoglio The reason for my solution is to have **one** shared place for input parameters without needing to write `[InlineData]` multiple times for multiple tests (this seem as duplicity for me). Do you insist on your solution?

Reply:
@deqenq That's a good point. As I thought more about it, I came to the conclusion that this doesn't need to be a parameterized test after all. A `TimeSpan` is a `TimeSpan`, regardless of how you construct it.

I suggest we remove the parameter altoghether and simply create a `TimeSpan` from a decimal number of seconds, as this mirrors the example used in [curl's documentation](https://ec.haxx.se/usingcurl-timeouts.html#maximum-time-allowed-to-spend).

Something like:

```csharp
// Given
var maxTime = TimeSpan.FromSeconds(2.37);
```

Reply:
@ecampidoglio Good idea, I will update the code this way.